django/publicmapping/redistricting/config.py

```diff
@@ -1603,22 +1630,6 @@ def feature_template(name, title=None, alias=None, attributes=[]):
                 'dataLinks': [],
                 'nativeCRS': 'EPSG:3785',
                 'srs': 'EPSG:3785',
-                # Set the bounding box to the maximum spherical mercator extent
-                # in order to avoid all issues with geowebcache tile offsets
-                'nativeBoundingBox': {
-                    'minx': '%0.1f' % -8974318.61690597,
-                    'miny': '%0.1f' % 4813698.29328726,
-                    'maxx': '%0.1f' % -8312679.70006949,
-                    'maxy': '%0.1f' % 5244191.63658937,
-                    'crs': 'EPSG:3785'
-                },
-                'latLonBoundingBox': {
-                    'minx': -80.61767578124999,
-                    'miny': 39.639537564366684,
-                    'maxx': -74.674072265625,
-                    'maxy': 42.5530802889558,
-                    'crs': 'EPSG:4326'
-                },
```

Comment:
Yeah, I have recollections of these being required in order to get zoom levels correct (or something along those lines). This might be related to why the tiles appear shifted. One other reason for that could be the slight differences among the competing SRIDs at the time (3785 vs. 3857 vs. 900913) -- that is something that we definitely ran into when developing this, and probably have a couple workarounds in place.

Reply:
Also interesting. I'll set the bbox to the max extent again and see if that fixes things. I'm rerunning the whole configuration after rebasing on develop currently anyway.

Reply:
Er, rather, I rebased on Derek's open PR